tests/common/exceptions.h

```diff
@@ -42,7 +42,7 @@ namespace sycl_cts {
 #define SYCL_CTS_SUPPORT_HAS_ERROR_CATEGORY_FOR 0
 #define SYCL_CTS_SUPPORT_HAS_MAKE_ERROR_CODE 0
 
-#elif defined(__DPCPP__)
+#elif defined(SYCL_IMPLEMENTATION_INTEL)
```

Comment:
Good to know this exists now!

Reply:
@bader Thanks, updated with `__SYCL_COMPILER_VERSION`